src/components/HistorySideBar.tsx

```diff
@@ -1,23 +1,24 @@
-import { PastCommitNode } from './PastCommitNode';
+import { PastCommitNode } from "./PastCommitNode";
 
-import { SingleCommitInfo } from '../git';
```

Comment:
This looks great! I'm starting to review this more however I'd love to see some Unit Tests for this component.  We already have some for the branch component so you could build off that.